docs/examples/general/index.rst

```diff
@@ -0,0 +1,13 @@
+General
+=======
+
+.. toctree::
+   :maxdepth: 2
+
+   custom_operator/create_a_custom_operator.ipynb
```

Comment:
Maybe custom operator and python_operator should go into separate category "Extending DALI"?

Reply:
:+1: